source/Tutorials/Advanced/Discovery-Server/scripts/super_client_configuration_file.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<dds>
+    <profiles xmlns="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles">
+        <participant profile_name="super_client_profile" is_default_profile="true">
+            <rtps>
+                <prefix>44.49.53.43.53.45.52.56.45.52.5f.30</prefix>
+                <builtin>
+                    <discovery_config>
+                        <discoveryProtocol>SUPER_CLIENT</discoveryProtocol>
+                        <discoveryServersList>
+                            <RemoteServer prefix="44.53.00.5f.45.50.52.4f.53.49.4d.41">
```

Comment:
I'd add a comment here stating that the prefix should be updated according to the server ID passed on the CLI. We somehow need a note which links to Fast DDS docs where we explain this

Reply:
Done